From 399aaa3b42c81626c18f6ece97e26e5e004a6254 Mon Sep 17 00:00:00 2001
From: Sam Ravnborg <srn@skov.dk>
Date: Sat, 18 Feb 2017 10:05:01 +0100
Subject: [PATCH] arm9-cpu: enable icache

This should allow faster boot - but no specific timings done
---
 board/skov/arm9-cpu/arm9-cpu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/skov/arm9-cpu/arm9-cpu.c b/board/skov/arm9-cpu/arm9-cpu.c
index e5a8fba..e4eed7d 100644
--- a/board/skov/arm9-cpu/arm9-cpu.c
+++ b/board/skov/arm9-cpu/arm9-cpu.c
@@ -488,7 +488,7 @@ void enable_caches(void)
 {
 	// Defining this function prevents a print during boot
 	// We may later enable icache her like this:
-	// icache_enable();
+	icache_enable();
 }
 
 int dram_init(void)
-- 
1.8.3.1

